diff --git a/certora/munged/syndicate/Syndicate.sol b/certora/munged/syndicate/Syndicate.sol
index 07f69c8..8feb948 100644
--- a/certora/munged/syndicate/Syndicate.sol
+++ b/certora/munged/syndicate/Syndicate.sol
@@ -213,7 +213,7 @@ contract Syndicate is ISyndicateInit, Initializable, Ownable, ReentrancyGuard, S
             blsKey _blsPubKey = _blsPubKeys[i];
             uint256 _sETHAmount = _sETHAmounts[i];
 
-            if (_sETHAmount < 1 gwei) revert FreeFloatingStakeAmountTooSmall();
+            //if (_sETHAmount < 1 gwei) revert FreeFloatingStakeAmountTooSmall();
             if (!isKnotRegistered[_blsPubKey] || isNoLongerPartOfSyndicate[_blsPubKey]) revert KnotIsNotRegisteredWithSyndicate();
 
             if (block.number < priorityStakingEndBlock && !isPriorityStaker[_onBehalfOf]) revert NotPriorityStaker();
